packages/synthetic-chain/src/cli/proposals.ts

```diff
@@ -42,6 +43,10 @@ function readInfo(proposalPath: string): ProposalInfo {
   };
 }
 
+export function encodeUpgradeInfo(upgradeInfo: unknown): string {
+  return upgradeInfo != null ? JSON.stringify(upgradeInfo) : '';
```

Comment:
Nit: swap the expressions so the condition isn't negated, or better yet no ternary

Reply:
Sorry, I forgot about this before publishing.